extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/sys/seal/VaultInit.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.vault.runtime.sys.seal;
+
+import java.util.List;
+
+public class VaultInit {
+
+    private List<String> keys;
+    private List<String> keysBase64;
+    private String rootToken;
+
+    public VaultInit(List<String> keys, List<String> keysBase64, String rootToken) {
+        this.keys = keys;
+        this.keysBase64 = keysBase64;
+        this.rootToken = rootToken;
+    }
+
+    public List<String> getKeys() {
+        return keys;
+    }
+
+    public List<String> getKeysBase64() {
+        return keysBase64;
+    }
+
+    public String getRootToken() {
+        return rootToken;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
it is too dangerous to include sensitive data into a `toString()`. it is way too easy for somebody to do a `log.info("init result = " + vaultInit)`. if we want to have a meaningful `toString()`, let's replace it with: 
```
return "VaultInit{"+keys.size()+" keys)"
```